### PR TITLE
Usagov 1425 create csv export of cms users roles

### PR DESCRIPTION
**Pull request by kyle-jester**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1425

## Type of Changes

- [x ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [x ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

### Requires New Config
- [ ] Yes
- [ ] No

### Requires New Content
- [ ] Yes
- [ x] No

### Validation Steps
- Once loaded you may need to flush the routing and links cache

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: config/sync/views.view.user_admin_people.yml [hunk 1]
```diff
@@ -22,56 +26,6 @@ display:
     display_options:
       title: People
       fields:
-        user_bulk_form:
```
Comment (akf) [inline config/sync/views.view.user_admin_people.yml]: I don't think you meant to remove this -- it's the checkbox that lets us block several users at once. It should be removed from the CSV display, but not from the form.

File: config/sync/views.view.user_admin_people.yml [hunk 2]
```diff
@@ -458,6 +412,70 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
+        mail_1:
```
Comment (akf) [inline config/sync/views.view.user_admin_people.yml]: I think it's good to have this field on both displays, but as long as you're in there, let's move the "Operations" field/column all the way to the right on the form.

**Review by akf — CHANGES_REQUESTED** [review]
Looks pretty good! I noticed that the checkbox is missing from the form now. 
Also, please add the "Created" date to the CSV export. It should be formatted there the same way "Last access" is (and labeled "Created" rather than "Member since"). 

In fact, now that we're mucking about with this form, I think everyone would probably rather see "Created" and a date on the form version of this too.

File: config/sync/views.view.user_admin_people.yml (hunk 1 above)
Comment (kyle-jester) [reply config/sync/views.view.user_admin_people.yml]: Correct , didnt mean to remove it from the form thought it was in the CSV, also adding  moving the other fields.

**Review by akf — APPROVED** [review]
Looks good, thank you!

**Pull request merged**